src/main/kotlin/solve/catalogue/model/CatalogueModel.kt

```diff
@@ -6,7 +6,12 @@ import solve.project.model.ProjectFrame
 
 enum class ViewFormat {
     FileName,
-    ImagePreview
+    ImagePreview;
+
+    fun differentFormat() = when (this) {
```

Comment:
don't use such funcs, if you need boolean use it, or use selection of neded type